src/vs/workbench/contrib/remote/browser/tunnelView.ts

```diff
@@ -666,11 +654,7 @@ MenuRegistry.appendMenuItem(MenuId.TunnelInline, ({
 	command: {
 		id: ClosePortAction.ID,
 		title: ClosePortAction.LABEL,
-		iconClassName: 'codicon-x',
-		iconLocation: {
-			dark: undefined,
-			light: undefined
-		}
+		icon: { id: 'codicon/x' }
```

Comment:
@alexr00 My change should impact this but I didn't find the `x` and `plus` icon on https://microsoft.github.io/vscode-codicons/dist/codicon.html. Is that site outdated or where do these icons come from?

Reply:
It looks like `codicon-plus` is the same as `codicon-add` and `codicon-x` is the same as `codicon-close`. I was getting them from https://github.com/Microsoft/vscode/blob/master/src/vs/base/browser/ui/codiconLabel/codicon/codicon.css

Reply:
Check - thanks for clarifying